media-plugins/vdr-osdteletext/vdr-osdteletext-0.9.7.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit user vdr-plugin-2
+
+VERSION="2161" # every bump, new version
+
+DESCRIPTION="VDR Plugin: Osd-Teletext displays the teletext on the OSD"
+HOMEPAGE="https://projects.vdr-developer.org/projects/plg-osdteletext"
+SRC_URI="mirror://vdr-developerorg/${VERSION}/${P}.tgz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~x86"
+
+DEPEND=">=media-video/vdr-1.7.39"
+RDEPEND="${DEPEND}"
+
+VDR_RCADDON_FILE="${FILESDIR}/rc-addon-v3.sh"
+VDR_CONFD_FILE="${FILESDIR}/confd-v2"
+
+QA_FLAGS_IGNORED="
+	usr/lib/vdr/plugins/libvdr-.*
+	usr/lib64/vdr/plugins/libvdr-.*"
+
+src_install() {
+	vdr-plugin-2_src_install
+
+	insinto /etc/sudoers.d
+	insopts -m440
+	newins "${FILESDIR}/vdr-osdteletext.sudo" vdr-osdteletext
+
+	local VDR_USER_HOME=$(egethome vdr)
```

Comment:
`local` variables are always written in smallcase. Rest seems good!

Reply:
ah... ok. changed it.

Reply:
replaced with `local vdr_user_home=$(egethome vdr)`